.gitlab/.gitlab-ci.yml

```diff
@@ -192,25 +192,42 @@ build fbsql amd64:
   script:
     - export SOURCE_DATE_EPOCH=$(git log -1 --pretty=%ct)
     - date
-    - GOOS="linux" GOARCH="amd64" make docker-build-fbsql BUILD_CGO=1
+    - GOOS="linux" GOARCH="amd64" make docker-build-fbsql
     - GOOS="darwin" GOARCH="amd64" make docker-build-fbsql
   artifacts:
     paths:
       - ./build/fbsql_*
 
-build fbsql arm64:
+build fbsql arm64 darwin:
```

Comment:
the `build fbsql amd64` step builds darwin amd64. 

I broke arm64 our because `Dockerfile-fbsql-linux` can and did run on `shell-arm64` but the `Dockerfile-fbsql-darwin` doesn't. It gives an exec error. I believe, based on a bit of research, this is because, if I set the `shell-arm64`, `Dockerfile-fbsql-linux` is able to use the `arm64` version of the golang container. But `Dockerfile-fbsql-darwin` depends on the customer image I built which is `amd64`. Take a look at `Dockerfile-darwin-cgo-builder`. There was no way to pull that `Xcode.xip` file from a public server so I built the image locally and pushed it to docker hub.

Happy to talk this one through.